src/status_im/ui2/screens/chat/components/new_chat/styles.cljs

```diff
@@ -13,4 +14,9 @@
    :margin-left     20
    :margin-bottom   36
    :justify-content :center
-   :align-items     :center})
\ No newline at end of file
+   :align-items     :center})
+
+(defn chat-button
+  [{:keys [bottom]}]
+  {:position :absolute
+   :bottom   (- bottom 50)})
```

Comment:
this looks like a hack, could you elaborate?

Reply:
but you've made the position absolute , so its not only y position its also z position